util/object_pool.c

```diff
@@ -1063,6 +1128,40 @@ static void objpool_rand_clear(comdb2_objpool_t op)
 }
 // ^^^^^^get/put impl
 
+static void objpool_evict_all_int(comdb2_objpool_t op)
```

Comment:
Don't need this.

Reply:
See comment above.  If we truly don't need it, that's cool.  I just need a way to make sure that all objects in the pool are emptied out (i.e. if they aren't in use).

Reply:
Gotcha. I misunderstood this. This is cool.